memstore/job_manager.go

```diff
@@ -42,11 +64,12 @@ type archiveJobManager struct {
 }
 
 // newArchiveJobManager creates a new jobManager to manage archive jobs.
-func newArchiveJobManager(scheduler *schedulerImpl) jobManager {
+func newArchiveJobManager(scheduler *schedulerImpl) JobManager {
 	return &archiveJobManager{
 		jobDetails: make(map[string]*ArchiveJobDetail),
 		memStore:   scheduler.memStore,
 		scheduler:  scheduler,
+		enabler:    enabler{enabled: false},
```

Comment:
the default value of bool is false, so there is no need to initialize enabler here